app/src/main/java/com/sun/music_66/extentions/ActivityExtension.kt

```diff
@@ -0,0 +1,43 @@
+package com.sun.music_66.extentions
+
+import android.app.Activity
+import android.os.Bundle
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+import com.sun.music_66.constant.ScreenAnimation
+import com.sun.music_66.base.view.BaseFragment
+
+fun Activity.openFragment(manager: FragmentManager,
+                          transaction: FragmentTransaction,
+                          newClass: Class<out BaseFragment>,
+                          screenAnimation: ScreenAnimation,
+                          data: Bundle?,
```

Comment:
truyền dữ liệu qua fragment hay activity nên tuần thủ rule 2.11
nên a nghĩ k cần thiết có bundle ở đây
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment